commands/info/index.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
+
+const Command = require("@lerna/command");
+const output = require("@lerna/output");
+const envinfo = require("envinfo");
+
+module.exports = factory;
+
+function factory(argv) {
+  return new InfoCommand(argv);
+}
+
+class InfoCommand extends Command {
+  execute() {
+    output("\n Environment info:");
```

Comment:
This strikes me as kinda redundant. I think we only need the actual output of `envinfo.run()`